n2y/page.py

```diff
@@ -101,7 +103,22 @@ def parent(self):
             return self.client.get_database(self.notion_parent["database_id"])
 
     def to_pandoc(self):
-        return self.block.to_pandoc()
```

Comment:
@matt-hayden ideally this would not go in the page-level block, but rather the table level block. Is there a reason it needs to be a the page level? Ideally we'll keep the page level pretty thin if we can.

Reply:
No, and if we don't need to know how many tables are affected in total, then this can all live in Table methods.